deployments/installer/install.sh

```diff
@@ -454,7 +458,7 @@ install() {
       ;;
   esac
 
-  configure_access_token "$access_token"
+  configure_access_token "$access_token" "$service_user" "$service_group"
```

Comment:
This should be moved to after `create_user_group` is called.

Reply:
Ok, I just moved all of these to after that.